ui/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>io.jenkins.plugins</groupId>
             <artifactId>jaxb</artifactId>
-            <version>2.2.11</version>
+            <version>2.3.x-20190131.120627-1</version>
```

Comment:
I guess you'll be replacing this by a release?

Reply:
I just re-read your description and it answers my question. Please ignore me. :man_facepalming:

Reply:
yes. But was stuck because of https://github.com/jenkinsci/jaxb-plugin/pull/6#issuecomment-459338867, but decided to yolo merge.

Will release as soon as I see the `master` branch of https://github.com/jenkinsci/jaxb-plugin/ go green :heavy_check_mark:.

Reply:
done